apps/web/src/components/FilterDialog/FilterDialog.tsx

```diff
@@ -15,12 +15,17 @@ import { notEmpty } from "@gc-digital-talent/helpers";
 // Used by specific dialogs
 export type CommonFilterDialogProps<TFieldValues extends FieldValues> = {
   onSubmit: SubmitHandler<TFieldValues>;
+  /** Defaults values, including from URL */
   defaultValues?: Partial<TFieldValues>;
+  /** Initial values to reset the form to, not including URL, */
+  initialValues: TFieldValues;
```

Comment:
Hmm, just thought of `pickedValues` or `chosenValues`?

Reply:
That makes more sense! Since it is technically the ones a user selected and not the ones we hard code as default. I'll quickly mention at stand up and see what people think.

Reply:
Okay, now that I've looked at the code, I wouldn't recommend `routeValues` 😅 . It should make sense more generally - those values don't _need_ to come from the route. Funnily enough, after understanding what's going on, I think "initialValues" (for routeValues) and "defaultValues" (for initialValues) actually make a lot of sense, but they're not helpful to _learn_ what's going on... :(

How about trying to be really explicit about how these values are used, instead of where they come from:
```
  /** When the user resets filters they will return to these values. If initialValues is empty, resetValues is used to initialize the filters. */
  resetValues: TFieldValues;
  /** If initialValues is set, it will override resetValues when the filter form is first initialized. */
  initialValues?: Partial<TFieldValues>;
```
Alternatively, `overrideValues` instead of `initialValues`?

Reply:
I liked that so I just copy pasted 😄 

Updated in [update filter state props for even more clarity](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8784/commits/e3fefe9464c9aef41195540dc84a3127e65fa51c)